src/Illuminate/Foundation/Console/stubs/event.stub

```diff
@@ -27,10 +25,12 @@ class {{ class }}
     /**
      * Get the channels the event should broadcast on.
      *
-     * @return \Illuminate\Broadcasting\Channel|array
+     * @return array<int, \Illuminate\Broadcasting\Channel>
      */
-    public function broadcastOn()
+    public function broadcastOn(): array
```

Comment:
I believe this should be a return type of `array|Channel` as the current `BroadcastEvent` still wraps the return of this method.

https://github.com/laravel/framework/blob/b856dc5e8bd65a4d9ef1afada998855b52388c30/src/Illuminate/Broadcasting/BroadcastEvent.php#L71